### PR TITLE
Added RecycleItemsWorker and replaced some code in SeenFortWorker

### PR DESCRIPTION
**Pull request by meyer9**
Added RecycleItemsWorker and replaced some code in SeenFortWorker

### CODE REVIEW

File: pokemongo_bot/cell_workers/seen_fort_worker.py [hunk 1]
```diff
@@ -56,11 +57,8 @@ def work(self):
                     logger.log("[+] " + str(item_count) + "x " + item_name,
                                "green")
 
-                    if str(item_id) in self.config.item_filter:
-                        logger.log("[+] Recycling " + str(item_count) + "x " + item_name + "...", 'green')
-
-                        # RECYCLE_INVENTORY_ITEM
-                        self.bot.drop_item(item_id=item_id, count=item_count)
+                recycle_worker = RecycleItemsWorker(self.bot)
```
Comment (jtdroste) [inline pokemongo_bot/cell_workers/seen_fort_worker.py]: I would wrap this inside an if it's enabled

File: pokemongo_bot/cell_workers/recycle_items_worker.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def __init__(self, bot):
+        self.api = bot.api
+        self.bot = bot
+        self.position = bot.position
+        self.config = bot.config
+        self.item_list = bot.item_list
+
+    def work(self):
+        self.bot.update_inventory()
+        if self.config.recycle_items:
```
Comment (jtdroste) [inline pokemongo_bot/cell_workers/recycle_items_worker.py]: I would remove this.  When a worker is told to work, it should do it.  Leave the responsibility of if this worker recycles or not up to the caller, not the function

**Comment by jtdroste** [comment]
Added my comments

**Comment by jtdroste** [comment]
Awesome

👍  

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/73/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
Well done

:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/73/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**